src/main/java/Duke.java

```diff
@@ -14,32 +14,49 @@ public static void main(String[] args) {
                 printLines();
                 System.out.println("Here are the tasks in your list:");
                 for (int i = 1; i <= task ; i++) {
-                    System.out.println(i + "." + "[" + list[i-1].getStatusIcon()
-                    + "] " + list[i-1].description);
+                    System.out.println(i + "." + list[i-1]);
                 }
                 printLines();
             }else if (command.contains("done")) {
                 String[] parts = command.split(" ");
                 int taskNo = Integer.parseInt(parts[1]) - 1 ;
-                list[taskNo].markAsDone();
-                printLines();
-                System.out.println("Nice! I've marked this task as done:");
-                System.out.println("[" + list[taskNo].getStatusIcon() + "] " + list[taskNo].description);
                 printLines();
-            } else {
-                Task t = new Task(command);
-                list[task] = t;
-                printLines();
-                System.out.println("added: "+ command);
+                list[taskNo].markAsDone();
                 printLines();
+            } else if (command.contains("deadline")) {
```

Comment:
It is not suggested to use so many if else if. I think you can try switch case.
switch (expression) {
  case x:
    // code block
    break;
  case y:
    // code block
    break;
  default:
    // code block
}

Reply:
Thanks for the recommendation! I have changed it to switch case in my latest repo